site/assets/scss/_sidebar.scss

```diff
@@ -74,9 +69,7 @@
     }
   }
 
-  > .active > a,
-  > .active:hover > a,
-  > .active:focus > a {
+  [aria-current="page"] {
```

Comment:
Same here, add an `active` class in the docs-sidebar template instead.

Reply:
Well, the idea was to reduce the custom JS needed for this thing. Ideally we should rely on the classes the collapse plugin is using, but worse case scenario, we add back an active class.

Reply:
This can be achieved by changing the Hugo template.

Reply:
Yeah, well, I'm not sure it's simpler, so feel free to provide a patch. BTW I still want to add the page's class in body, but with our templates being messed up, it wasn't so easy last time I tried.

Reply:
@XhmikosR There is already a `.active` class on the parent in Hugo template, let's use it. I just move it to the link, since it's not used on the list item anyway…